Interface extension in subprocess.ts conflict causes flowcrypt-account build to fail

flowcrypt-account fails to build because the `exitCode` interface defined in flowcrypt-node-modules/source/subprocess.ts conflicts with an already existing interface provided by @type/node/child_process.d.ts. 

Here is the error occurring on Semaphore:

```node
../flowcrypt-node-modules/source/subprocess.ts:4:18 - error TS2430: 
Interface 
  'import("/home/semaphore/flowcrypt-node-modules/source/subprocess").ChildProcess' 
incorrectly extends interface 
  'import("child_process").ChildProcess'.

Types of property 'exitCode' are incompatible.
  Type 'number | null | undefined' is not assignable to type 'number | null'.
    Type 'undefined' is not assignable to type 'number | null'.
```

Here is the interface as we define it:

https://github.com/FlowCrypt/flowcrypt-node-modules/blob/1601e0db3334c560bb1cbd1f9cdfce397b5e36af/source/subprocess.ts#L4-L6

This interface used to work because, as the comment notes, up until May 12th (ten days ago) @type/node/child_process.d.ts lacked a definition for `exitCode`. However, that is no longer the case. A recent commit has added the definition to the interface:

```typescript
readonly exitCode: number | null;
```
(from [DefinitelyTyped/types/node/child_process.d.ts#L25](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6fd3e189fa73b23429e60f0176532da6a11c7bfb/types/node/child_process.d.ts#L25))

Note that, unlike our definition, this one is *not* optional. That's the surface-level cause of the error as seen on Semaphore: according to the official type definition, `exitCode` must be either `number` or `null`, whereas our choice to make it optional means it could also be undefined. 

This actually doesn't fail on my local machine, because when I install it still grabs the older version of @types/node. However, on semaphore this is not the case.

 ```bash
semaphore@semaphore-vm:~/flowcrypt-account$ grep exitCode node_modules/@types/node/child_process.d.ts 
        readonly exitCode: number | null;
```

We could fix this in flowcrypt-account by pegging @types/node to an older version that still lacks the interface. However, that strikes me as obviously inferior to simply removing the interface extension altogether in subprocess.ts now that it is no longer necessary.

It's interesting that this flaw in the type bindings for Node remained for so long. It was originally reported as early as 2016 in https://github.com/microsoft/TypeScript/issues/9940. 

I'll have a PR up momentarily!

--- a/source/subprocess.ts
+++ b/source/subprocess.ts
@@ -1,11 +1,7 @@
 import * as child_process from 'child_process';
 import * as setNodeCleanupCb from 'node-cleanup';
 
-export interface ChildProcess extends child_process.ChildProcess {
-  exitCode?: number | null; // ts is missing this in def
-}
-
-const PROCESSES: ChildProcess[] = [];
+const PROCESSES: child_process.ChildProcess[] = [];
 const SPAWN_READINESS_TIMEOUT = 10 * 1000;
 
 export class SubprocessError extends Error {
@@ -30,10 +26,10 @@ export class Subprocess {
    */
   public static onStderr = (r: { stderr: Buffer, cmd: string, args: string[] }): void => undefined;
 
-  public static spawn = (cmd: string, rawArgs: (string | number)[], readiness_indicator?: string, env?: { [k: string]: string }): Promise<ChildProcess> => new Promise((resolve, reject) => {
+  public static spawn = (cmd: string, rawArgs: (string | number)[], readiness_indicator?: string, env?: { [k: string]: string }): Promise<child_process.ChildProcess> => new Promise((resolve, reject) => {
     let ready = false;
     const args = rawArgs.map(String);
-    const p: ChildProcess = child_process.spawn(cmd, args, { env });
+    const p: child_process.ChildProcess = child_process.spawn(cmd, args, { env });
     PROCESSES.push(p);
     if (p.stdout) {
       p.stdout.on('data', (stdout: Buffer) => {

